randovania/data/json_data/prime3.txt

```diff
@@ -3899,25 +4763,53 @@ Asset id: 13589229454537797972
 
 > Main Room; Heals? False
   > Dock to Transit Station 2-A
-      HyperModeTank and Hyper Grapple
+      After Command Station Phazon Blob
   > Dock to Defense Access (Base)
       Trivial
-  > Morph Ball Tunnels
+  > Pickup (Missile Expansion)
+      After Command Station Morph Ball Barrier
+  > Event - Morph Ball Barrier
       Grapple Voltage
+  > Event - Phazon Blob
+      HyperModeTank and Hyper Grapple
+  > Event - Disable Defense
+      Morph Ball Bombs and Shoot Phazite (X-Ray and Nova)
```

Comment:
Move all these requirements to Defense Access Base door node, and delete Main room node